src/template.c

```diff
@@ -5202,7 +5202,12 @@ MATCH TemplateValueParameter::matchArg(Scope *sc, RootObject *oarg,
             ei = new VarExp(loc, f);
             ei = ei->semantic(sc);
             if (!f->needThis())
-                ei = resolvePropertiesOnly(sc, ei);
+            {
+                unsigned int errors = global.startGagging();
+                ei = resolveProperties(sc, ei);
+                if (global.endGagging(errors))
```

Comment:
I'll pull this for now, but I'd really like to get away from using global error counts and instead rely on error propagation.